### PR TITLE
[JENKINS-40433] Increase test coverage for credentials plugin

### PR DESCRIPTION
**Pull request by varyvol**
[JENKINS-40433](https://issues.jenkins-ci.org/browse/JENKINS-40433)

- Test credentials for default domain as well as custom domains.
- Test personal credentials.

@reviewbybees

### CODE REVIEW

File: src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/DomainPage.java [hunk 1]
```diff
@@ -28,7 +32,19 @@ public URL getConfigUrl() {
...
+        } else {
+            clickButton("OK");
+        }
+
         assertThat(driver, not(hasContent("This page expects a form submission")));
     }
+
+    public void delete() {
+        visit(url("delete"));
+        elasticSleep(1000);
```
Comment (raul-arabaolaza) [inline src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/DomainPage.java]: Better use a `waitFor`

Comment (varyvol) [inline src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/DomainPage.java]: Done.

File: src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/DomainPage.java [hunk 2]
```diff
@@ -28,7 +32,19 @@ public URL getConfigUrl() {
 
     @Override
     public void save() {
-        clickButton("OK");
+        if (driver.getCurrentUrl().contains("configure")) {
```
Comment (varyvol) [inline src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/DomainPage.java]: As discussed offline, this is not the Jenkins configuration page but the domain one. Anyway I am improving the detection of such configuration page by checking the whole URL.

**Comment by mafraba** [comment]
lgtm 🐝

**Comment by raul-arabaolaza** [comment]
:bee: granted the CI build finishes satisfactory

File: src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/CredentialsPage.java [hunk 3]
```diff
@@ -60,7 +70,7 @@ public WebDriver open() {
         WebDriver wd = super.open();
         // wait for default form fields to be present to avoid possible race
         // condition when changing credentials type too fast (happens rarely)
-        waitFor(by.name("_.scope"));
+        waitFor(by.name("_.id"));
```
Comment (varyvol) [inline src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/CredentialsPage.java]: This changes the field to wait for (introduced in [PR #243](https://github.com/jenkinsci/acceptance-test-harness/pull/243) as personal user credentials does not have scope.

**Comment by varyvol** [comment]
Build and tests passing working correctly.

**Pull request merged**